consumers/campaigner/main.go

```diff
@@ -36,14 +37,23 @@ func (h *MessageHandler) HandleMessage(m *nsq.Message) error {
 		return nil
 	}
 
-	msg := new(entities.SendCampaignParams)
+	msg := new(entities.CampaignerMessageBody)
```

Comment:
Let's be consistent in naming the struct for nsq messages. We can do something like: `<TopicName>TopicParams`, or in this case it will be `CampaignerTopicParams`.